src/api/index.js

```diff
@@ -10,17 +10,13 @@ exports.get =  function(requesturl="", headers={}, callback){
   })
   .then(function(res){
     data_ = res.data;
-    console.log(res);
     return callback(error_, res);
   })
   .catch(function (error) {
     error_ = error;
     return callback(error_);
   })
   .finally(function () {
-    // always executed
-    console.log("Request done");
-  //  return callback(error_, data_);
   });
```

Comment:
Finally block is empty and not being used. No need to finally as catch will "catch" the error